packages/devtools_app/lib/src/framework/status_line.dart

```diff
@@ -46,23 +46,27 @@ class StatusLine extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
+    final color = isConnected ? theme.colorScheme.onPrimary : null;
     final height = statusLineHeight + padding.top + padding.bottom;
     return ValueListenableBuilder<bool>(
       valueListenable: currentScreen.showIsolateSelector,
       builder: (context, showIsolateSelector, _) {
-        return Container(
-          decoration: BoxDecoration(
-            color: isConnected ? theme.colorScheme.primary : null,
-            border: Border(
-              top: Divider.createBorderSide(context, width: 1.0),
+        return DefaultTextStyle.merge(
+          style: TextStyle(color: color),
```

Comment:
I'm surprised that the color we set in `_getStatusItems` is not sufficient here. Do we no longer need to set the color of all the items individually in `_getStatusItems`

Reply:
I chose this since it looked like the Text Widget wasn't picking up the style in the network screen.
https://github.com/Flutter/devtools/blob/master/packages/devtools_app/lib/src/screens/network/network_screen.dart#L59

I could have done it lower down in the network screen but figured this would be better in case there is another Text widget in the future.